src/api/getLanglinks.js

```diff
@@ -6,14 +6,14 @@ export const getLanglinks = (lang, title) => {
     titles: title,
     prop: 'langlinks',
     lllimit: 500,
-    llprop: 'langname'
+    llprop: 'langname|autonym'
   }
   const url = buildMwApiUrl(lang, params)
   return cachedFetch(url, response => {
     const { pages } = response.query
     return pages[0].langlinks.map(item => (
       {
-        title: item.langname,
+        title: [item.langname, item.autonym],
         lang: item.lang,
         description: item.title
```

Comment:
I would suggest renaming all the key to `langname` `autonym` or other sensible names, then parse it to ListView in the form of `{ title, lang, description}` (this is used by ListView Component)